state/allwatcher_internal_test.go

```diff
@@ -1882,6 +1892,13 @@ func (s *allModelWatcherStateSuite) TestStateWatcher(c *gc.C) {
 			HasVote:   false,
 			WantsVote: false,
 		},
+	}, {
+		Entity: &multiwatcher.GenerationInfo{
+			ModelUUID:     st1.ModelUUID(),
+			Id:            "0",
```

Comment:
What is the `Id` from?

Reply:
The document itself gets a generated ID much like machines do, but none of the other logic has required it be exposed so far.